CMake configuration issue, now with SPIRV-Tools-static

Probably same thing as #6244, now with a different target through
```
CMake Error in .cache/cpm/zlib/1.3.1.2/CMakeLists.txt:
  The custom command generating

    /Users/chris/Documents/eden/build/_deps/spirv-tools-build/build-version.inc

  is attached to multiple targets:

    spirv-tools-build-version
    SPIRV-Tools-static

  but none of these is a common dependency of the other(s).  This is not
  allowed by the Xcode "new build system".
```

--- a/source/CMakeLists.txt
+++ b/source/CMakeLists.txt
@@ -149,11 +149,6 @@ add_custom_command(OUTPUT ${SPIRV_TOOLS_BUILD_VERSION_INC}
    DEPENDS ${SPIRV_TOOLS_BUILD_VERSION_INC_GENERATOR}
            ${SPIRV_TOOLS_CHANGES_FILE}
    COMMENT "Update build-version.inc in the SPIRV-Tools build directory (if necessary).")
-# Convenience target for standalone generation of the build-version.inc file.
-# This is not required for any dependence chain.
-add_custom_target(spirv-tools-build-version
-   DEPENDS ${SPIRV_TOOLS_BUILD_VERSION_INC})
-set_property(TARGET spirv-tools-build-version PROPERTY FOLDER "SPIRV-Tools build")
 
 list(APPEND PCH_DEPENDS
 	${CORE_TABLES_HEADER_INC_FILE}
@@ -340,7 +335,7 @@ function(spirv_tools_default_target_options target)
   )
   set_property(TARGET ${target} PROPERTY FOLDER "SPIRV-Tools libraries")
   spvtools_check_symbol_exports(${target})
-  add_dependencies(${target} spirv-tools-build-version core_tables extinst_tables)
+  add_dependencies(${target} core_tables extinst_tables)
 endfunction()
 
 # Always build ${SPIRV_TOOLS}-shared. This is expected distro packages, and

